Header header
float64 distance
uint8 robot1_id
uint8 robot2_id
float64 storage_disk
float64 communication_freq
   